core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/IntelliJProviderTest.scala

```diff
@@ -11,13 +11,18 @@ import org.junit.runners.JUnit4
 
 import org.virtuslab.ideprobe.Config
 import org.virtuslab.ideprobe.Extensions._
+import org.virtuslab.ideprobe.IdeProbePaths
 import org.virtuslab.ideprobe.IntelliJFixture
 import org.virtuslab.ideprobe.ide.intellij.IntelliJProvider
 
 @RunWith(classOf[JUnit4])
 final class IntelliJProviderTest {
   private implicit val ec: ExecutionContext = ExecutionContext.fromExecutorService(Executors.newCachedThreadPool())
 
+  private val probeConfig = IntelliJFixture.defaultIdeProbeConfig
+  private val defaultIntellijProvider = IntelliJProvider
```

Comment:
isn't that just IntelliJProvider.Default

Reply:
yes, changed to IntelliJProvider.Default